include/tvc.h

```diff
@@ -75,7 +77,6 @@ extern int __LIB__ asm_tvc_kbd_status();
 extern int __LIB__ getk();
```

Comment:
I don't understand why this is here since it's already defined in stdio.h

Reply:
removed